### PR TITLE
R4 Patient Resource: Preferred contact extension

### PR DESCRIPTION
**Pull request by viktorhegedusepam**
Description
----

Screenshot 2021-08-10 at 8 30 10
Screenshot 2021-08-10 at 8 30 56
Screenshot 2021-08-10 at 8 31 56
Screenshot 2021-08-10 at 8 32 20
Screenshot 2021-08-10 at 8 32 35

Changes in documentation & response.

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_patient.rb [hunk 1]
```diff
@@ -67,6 +67,76 @@ module Resources
...
       },
       'extension': [
+        {
+          'id': '59434424',
+          'extension': [
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/codeSet/4640016',
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_patient.rb:77]: Update the tenant id in all examples to "ec2458f2-1e24-41c8-b71b-0e701af7583d" which is the public sandbox. Ideally the examples that we give here should be reproducible. For now update the tenant id

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]

File: lib/resources/example_json/r4_examples_patient.rb (hunk 1 above)
Comment (mke9pndr6) [reply lib/resources/example_json/r4_examples_patient.rb]: updated

**Review by aki-mathur — APPROVED** [review]

**Comment by aki-mathur** [comment]
@viktorhegedusepam Can you fix the lint issue as CI build is failing

**Review by ashu-bdvt — APPROVED** [review]

**Comment by mke9pndr6** [comment]
Changes after changing tenant_id (all 3 types):
1. communication-type
Screenshot 2021-12-01 at 9 26 30

2. contact-method
Screenshot 2021-12-01 at 9 26 01

3. verified-datetime
Screenshot 2021-12-01 at 9 26 11

**Review by MTroianov — APPROVED** [review]

**Comment by aki-mathur** [comment]
Screenshot 2021-12-02 at 6 07 54 PM
Screenshot 2021-12-02 at 6 08 32 PM

**Pull request merged**